<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWindow</class>
 <widget class="QMainWindow" name="CMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Craps Simulator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="rollsLabel">
        <property name="text">
         <string>Number of Rolls: 0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="start">
        <property name="text">
         <string>Start</string>
        </property>
        <property name="icon">
         <iconset resource="application.qrc">
          <normaloff>:/resources/start.png</normaloff>:/resources/start.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="stop">
        <property name="text">
         <string>Stop</string>
        </property>
        <property name="icon">
         <iconset resource="application.qrc">
          <normaloff>:/resources/stop.png</normaloff>:/resources/stop.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="clear">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="application.qrc">
          <normaloff>:/resources/clear.png</normaloff>:/resources/clear.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="refresh">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="application.qrc">
          <normaloff>:/resources/refresh.png</normaloff>:/resources/refresh.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QCheckBox" name="simulateDice">
        <property name="text">
         <string>Simulate Dice?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Refresh Every # of Rolls:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="refreshRate">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000000</number>
        </property>
        <property name="value">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Stop after # of Rolls:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="stopRolls">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100000000</number>
        </property>
        <property name="value">
         <number>100000</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="rollsPerSec">
        <property name="text">
         <string>Rolls/Sec: 0</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>167</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QGroupBox" name="playStyle">
      <property name="title">
       <string>Play Style - All bets are played for even multiples on payout</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QCheckBox" name="playPass">
         <property name="text">
          <string>Play Pass Line?</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="4">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Base Bet:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="baseBet">
           <property name="minimum">
            <number>5</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="singleStep">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Odds Mulitplier Bet:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="oddsBet">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>2</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="playDontPass">
         <property name="text">
          <string>Play Don't Pass Line?</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QRadioButton" name="playInnerNumbers">
         <property name="text">
          <string>Play Inner Numbers (6/8)?</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QRadioButton" name="playBoard">
         <property name="text">
          <string>Play the Board (4/5/6/8/9)?</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QRadioButton" name="playInsideNumbers">
         <property name="text">
          <string>Play Inside Numbers (5/6/8/9)?</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="currentCash">
        <property name="text">
         <string>Current Cash Level: 0</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="resetOnGo">
      <property name="text">
       <string>Reset on Go?</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QCheckBox" name="logRolls">
      <property name="text">
       <string>Log Rolls?</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="0" colspan="3">
     <widget class="QPlainTextEdit" name="log">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>614</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Run</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionDice"/>
    <addaction name="actionTable"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRun"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="iconText">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Alt+F4</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/resources/start.png</normaloff>:/resources/start.png</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/resources/stop.png</normaloff>:/resources/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/resources/clear.png</normaloff>:/resources/clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionDice">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dice</string>
   </property>
  </action>
  <action name="actionTable">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Table</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>start</tabstop>
  <tabstop>stop</tabstop>
  <tabstop>clear</tabstop>
  <tabstop>refresh</tabstop>
  <tabstop>baseBet</tabstop>
  <tabstop>oddsBet</tabstop>
  <tabstop>playPass</tabstop>
  <tabstop>playInnerNumbers</tabstop>
  <tabstop>playBoard</tabstop>
  <tabstop>playDontPass</tabstop>
  <tabstop>playInsideNumbers</tabstop>
  <tabstop>resetOnGo</tabstop>
  <tabstop>logRolls</tabstop>
  <tabstop>log</tabstop>
 </tabstops>
 <resources>
  <include location="application.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>CMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>500</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
